src/plot/series/deck-gl-wrapper.js

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {PropTypes, Component} from 'react';
+import DeckGL, {OrthographicViewport} from 'deck.gl';
+
+import {ANIMATED_SERIES_PROPS} from 'utils/series-utils';
+import Animation from 'animation';
+
+class DeckGLWrapper extends Component {
+  render() {
+    const {
+      animation,
+      children,
+      marginLeft,
+      marginTop,
+      marginBottom,
+      marginRight,
+      innerHeight,
+      innerWidth,
+      _renderKey
+    } = this.props;
+
+    if (animation) {
```

Comment:
the crucial question now is how to make this section animate the layers correctly! am very confused